config/mv_checks.xml

```diff
@@ -455,7 +454,7 @@
             <property name="max" value="30"/>
         </module>
         <module name="LambdaBodyLength">
-            <property name="max" value="10"/>
+            <property name="max" value="50"/>
```

Comment:
Dang, 50 is HUGE. I'm okay with increasing this but maybe not quite this much. If a lamda is that big it should probably be broken up with some method calls or something.